lib/daru/view.rb

```diff
@@ -112,7 +112,10 @@ def dependent_script(lib=:nyaplot)
         when :nyaplot
           Nyaplot.init_script
         when :highcharts
-          LazyHighCharts.init_script
+          init_code = ''
```

Comment:
We must have one class method, which will return `init_css` and `init_script`.
So we can use the same class method [here](https://github.com/SciRuby/daru-view/pull/93/files#diff-6c740d124ea3031ad65d99a0e1baeff3R60) as well.
DRY and Loose Coupling concepts should be applied.